### PR TITLE
feat(duffel-payments): add ability to request and create refunds

### PR DESCRIPTION
**Pull request by lapa182**
Adds support for [refunds](https://duffel.com/docs/api/refunds)

### CODE REVIEW

File: src/DuffelPayments/Refunds/Refunds.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  public get = async (id: string): Promise<DuffelResponse<Refund>> =>
+    this.request({ method: 'GET', path: `${this.path}/${id}` })
+
+  /**
+   * Create a Refund to refund some money to a customer that they paid using a Payment Intent. You must specify the amount and currency. The currency is currently limited to the currency in which the Payment Intent was made.
+   * @param {Object.<CreateFund>} params - Endpoint params (amount, currency and payment_intent_id)
+   */
+  public create = async (
+    params: CreateRefund
+  ): Promise<DuffelResponse<any>> => {
```
Comment (go1t) [inline src/DuffelPayments/Refunds/Refunds.ts:30]: This seems to come with a response right? Should we type it accordingly? 
image

Comment (lapa182) [reply src/DuffelPayments/Refunds/Refunds.ts:30]: Ah, I missed this one! I put any just in the beginning and forgot about it. Thanks for noticing! Will add a fix shortly.

Comment (lapa182) [reply src/DuffelPayments/Refunds/Refunds.ts]: Done!

**Review by go1t — APPROVED** [review]
LGTM!

**Pull request merged**